ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -904,6 +904,52 @@ def test_paddle__len__(
     )
 
 
+# __matmul__
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="__matmul__",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    transpose_x=st.booleans(),
+    transpose_y=st.booleans(),
+    test_with_out=st.just(False),
```

Comment:
this parameter test_with_out doesn't exist in the decorator, please remove this.

Reply:
![image](https://github.com/unifyai/ivy/assets/141853536/f3e442a1-0a07-4eeb-8c46-30513a7e41b2)

Reply:
But I don't see these functions implemented yet.
![image](https://github.com/unifyai/ivy/assets/141853536/9e4c50be-fccc-4745-ab75-03264b4d86ff)
https://github.com/unifyai/ivy/issues/27759
The function you reserved by linking it to an open issue is still pending, so you can continue working on it until you close your PR.

Reply:
instance methods and dunder methods are different,
![image](https://github.com/unifyai/ivy/assets/141853536/294c42f4-2253-47c1-bbc4-978394fe3337)